app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_posts_counter
+    user.increment!(:posts_counter)
+  end
```

Comment:
- [ ] Let's change the `user` on line `7` to `author` since a post belongs to `author` in the `User` class. Remember that a post has a reference to user class through `author`. You can have something like this;

```ruby
def update_posts_counter
  author.increment!(:posts_counter)
end
```